misc/default.yml

```diff
@@ -75,6 +95,10 @@ platforms:
     gcc:
       features:
       defines:
+        - __AVR__
```

Comment:
I moved a definition of `__AVR__` into the ruby code so it's applied to all boards... was that a mistake on my part?

Reply:
I think so, __AVR__ is normally defined by avr-gcc, so only for the AVR-based Arduino boards. To emulate other boards (e.g. SAMD / Arduino Zero), I think __AVR__ should not be defined, otherwise libraries etc. will not be genuinely emulated and if they test for __AVR__ they will erronously assume that all kind of AVR-specific stuff is available.

Reply:
aah. Opened https://github.com/Arduino-CI/arduino_ci/issues/202 to track this.